.changes/v3.6.0/817-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Fix bug regarding `getCatalogByFilter` function [GH-817]
```

Comment:
Please put a more complete issue description as you have in PR summary.

Reply:
If I understand correctly, I would not add a changelog message at all, as this issue was introduced by PR which is in this release.

Reply:
We add changelog entries from govcd perspective - not Terraform, as govcd is used by other users too.

But, if this issue was introduced in this release, then this shouldn't need an entry at all, as @vbauzysvmware indicated.